docs/blog/everything_you_should_know_about_minecraft.md

```diff
@@ -0,0 +1,182 @@
+---
+sidebar: false
+title: 📓 新手玩家必修第一课
+date: 2024-07-31 15:30:00
+editor: bingling-sama
+---
+
+# 📓新手玩家必修第一课
+
+Mod、插件、光影……各种名词傻傻分不清？一篇文章带你搞懂 Minecraft 的一切。
+
+---
+
+*你可曾记得，那个记忆中逐渐模糊的方块世界，那是我们终将逝去的童年。*
+
+## 万物起源
+
+2009 年 5 月 17 日，Markus Persson 以 *Notch* 之名，发布了 Minecraft 的第一个版本，创造了无数玩家梦开始的地方。
+
+十五年后的今天，我们回首往事，一切似乎都发生在一瞬间。我们和这款游戏一起经历了各种社区内容的兴衰，太多美好的回忆历历在目。
+
+也许你是一个从未了解过这款游戏的新玩家，也许你是一直关注实况主们视频的云玩家，也许你是一步一步摸索到今天的骨灰级玩家；但我们今天，一起来聊聊与 Minecraft 有关的一切。
+
+~~如果你尝试过通过 Minecraft Wiki 来获取关于 Minecraft 的基础信息的话，你可能会觉得其内容过于枯燥，本文将以较为浅显的新玩家视角为你介绍有关 Minecraft 的各种概念。~~
+
+## Minecraft，启动！
+
+Minecraft 最初是一款使用 Java 编程语言开发的 PC 端单机游戏，随着时间的发展，衍生出了许多不同的版本，但我们今天要讨论的，是运行在 PC 上的 Minecraft Java Edition，即众人熟知的 Java 版。
+
+2014 年 9 月 15 日，微软以 25 亿美元的天价将 Minecraft 收入麾下。自此以后，玩家们在购买了 Minecraft 正版账号以后，就可以使用微软官方推出的 *Minecraft Launcher* （即 *Minecraft 启动器* ）来下载并游玩 Minecraft。
+
+由于官方启动器没有包含任何社区内容，这就催生了许多第三方 Minecraft 启动器。例如旧朝遗老 *MultiMC* 以及新生代的 *PolyMC*、*Prism Launcher* 等。
+
+然而对于国内玩家来说，记忆中更多的可能还是 *忘却的旋律*、*HMCL* 等第三方启动器。
+
+这些启动器各有特色，都兼容了许多有意思的社区内容，包括后面会提到的 *Mod*、*光影* 等。
+
+## Java 是个啥？咖啡吗？
+
+Minecraft 的游戏本体实际上是一个 `.jar` 格式的文件，它需要运行在 Java 虚拟机上。
+
+Java 是一门高级程序设计语言，属于运行时语言。所谓 *运行时（Runtime）*，就是程序在运行时所需要的环境。对于 Java 来说，这个运行时就是 Java 虚拟机。
+
+在游玩 Minecraft 之前，你需要安装游戏版本对应的 Java 版本。一般来说，安装 JRE 版本的 Java 就可以了，但也可能有其他原因会导致 JRE 无法完美运行游戏，所以会需要安装 JDK 版本的 Java。
+
+> JRE：Java Runtime Environment，Java 运行时环境，包含运行 Java 程序所需的程序内容
+> JDK：Java Development Kit，Java 开发工具包，包含 JRE 和其他编译、调试 Java 程序所需的工具
+
+关于各个游戏版本所需要的 Java 版本，请参阅下表：
+
+| Minecraft 版本 | 推荐 Java 版本   | 最低 Java 版本 |
+| ------------ | ------------ | ---------- |
+| 1.20.5 及以上    | Java 21 或更高  | Java 21    |
+| 1.18~1.20.4  | Java 17 或更高  | Java 17    |
+| 1.17         | Java 16 或更高  | Java 16    |
+| 1.9~1.16     | Java 8 或更高   | Java 8     |
+| 1.8          | Java 7 或更高   | Java 7     |
+| 1.6/1.7      | Java 6 或更高   | Java 6     |
+| 1.5          | Java 5 或更高   | Java 5     |
+| 1.4          | Java 1.4 或更高 | Java 1.4   |
+
+**注意：**
+
+* 以上表格仅供参考，实际情况可能会有所不同。
+* 建议使用最新版本的 Java，以获得最佳性能和安全保障。
+* 在某些改装版本中对 Java 版本可能有特殊要求，最好使用官方明确表示支持或建议使用的 Java 版本。
+* 某些旧版本 Minecraft 可能需要更早的 Java 版本，请参考官方网站或相关论坛的资料。
```

Comment:
```suggestion
- 以上表格仅供参考，实际情况可能会有所不同。
- 建议使用最新版本的 Java，以获得最佳性能和安全保障。
- 在某些改装版本中对 Java 版本可能有特殊要求，最好使用官方明确表示支持或建议使用的 Java 版本。
- 某些旧版本 Minecraft 可能需要更早的 Java 版本，请参考官方网站或相关论坛的资料。
```
Please use `-` instead of `*`.